src/main/resources/db/migration/V22__opprettet_tidspunkt.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE fakturaserie
+    ALTER COLUMN opprettet_tidspunkt TYPE TIMESTAMP,
+    ALTER COLUMN opprettet_tidspunkt SET NOT NULL;
```

Comment:
Finnes det ingen rader som ikke har opprettet_tidspunkt?

Reply:
Nei, alle rader har opprettet_tidspunkt som DATE. Men dette var ikke satt som NOT NULL og vi trenger TIMESTAMP

Reply:
Trengs ingen migrering for de feltene som allerede har en DATE fylt inn?

Reply:
De får en default